src/application.php

```diff
@@ -45,6 +49,13 @@
         }
 
         $configuration = readConfiguration($inputFile);
+
+        if (!$input->getOption('ignore-override') && file_exists($overrideFile)) {
+            $override = readConfiguration($overrideFile);
+
+            $configuration = array_merge_recursive($configuration, $override);
```

Comment:
@jubianchi 
I guess it's not that simple. Some properties should be overwritten (i.e. image definition), some should be merged (i.e. list of links), others - replaced..

I'm working on merging multiple docker-compose.yml files into the big one - that can be used here as input. 
So far it's developed internally, but I'll let you know as soon as it's published on github.